src/Components/Notifications/Notification.tsx

```diff
@@ -31,6 +32,7 @@ export const SUPPORTED_NOTIFICATION_TYPES = [
   "ARTICLE_FEATURED_ARTIST",
   "PARTNER_OFFER_CREATED",
   "VIEWING_ROOM_PUBLISHED",
+  "PARTNER_SHOW_OPENED",
```

Comment:
We haven't released `PARTNER_SHOW_OPENED` yet in production, so I don't think we need an extra feature flag.